data_aug/contrastive_learning_dataset.py

```diff
@@ -78,7 +81,17 @@ def get_dataset(self, name, n_views):
                                                           transform=ContrastiveLearningViewGenerator(
                                                               self.get_simclr_pipeline_transform(32),
                                                               n_views),
-                                                          download=True)
+                                                          download=True),
+                            'imagenet10': lambda: datasets.ImageNet('/shared/sets/datasets/vision/ImageNet', split='train', transform=
+                                                            transforms.Compose([transforms.Resize((224,224)),
+                                                            ContrastiveLearningViewGenerator(
+                                                              self.get_simclr_pipeline_transform(224),
+                                                              n_views)])),
+                            'imagenetdogs': lambda: datasets.ImageNet('/shared/sets/datasets/vision/ImageNet', split='train', transform=
```

Comment:
Hej, jaka jest roznica miedzy imagnet10 a imagenetdogs to mapowanie jest chyba identyczne?